docs/_sass/components/_article-info.scss

```diff
@@ -6,10 +6,31 @@
     @include media-breakpoint-down(md) {
       float: none;
     }
+    & > .edit-on-github-btn {
+      display: inline-block;
+      font-size: 16px;
+      font-weight: 500;
+      font-family: 'Inter';
```

Comment:
We don't have this font on the website. Let's use the same style as we have for the text on the right.
```css
font-size: 0.85rem;
font-weight: 300;
font-family: Roboto, sans-serif;
```